apps/prairielearn/src/ee/lib/ai-grading/ai-grading-util.ts

```diff
@@ -242,17 +249,19 @@ function generateSubmissionMessage({
       }
     });
 
-  if (submissionTextSegment) {
+  if (submissionTextSegment && !images_only) {
     message_content.push({
       type: 'text',
       text: submissionTextSegment,
     });
   }
 
-  message_content.push({
-    type: 'text',
-    text: '\n</response>\nHow would you grade this? Please return the JSON object.',
-  });
+  if (is_grading && !images_only) {
```

Comment:
This guard doesn't line up with the one on 191, could that cause mismatched `<response>` tags?